File: c/Lista2/Desafio7.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    float A, B, soma, resultado;
    
    printf("Digite o valor de A: ");
    scanf("%f", &A);

    printf("Digite o valor de B: ");
    scanf("%f", &B);

    soma = A+B;
    resultado = (soma * 45) / 100;
    printf("O resultado da operação é: %3.0f",resultado);

}

File: js/lista2/Desafio7.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio7- Faça um algoritmo para ler duas variáveis:

Apresentar 45% da soma destas.
*/

rl.question('Digite o valor de A: ', (a) => {
    rl.question('Digite o valor de B: ', (b) => {
      
    let valorFinal = (parseInt(a) + parseInt(b)) * 0.45;
        
        console.log(`A porcentagem dos dois números é: ${valorFinal}`);
      rl.close();
  });
});